mesh/src/main/java/no/nordicsemi/android/mesh/sensorutils/Power.java

```diff
@@ -0,0 +1,54 @@
+package no.nordicsemi.android.mesh.sensorutils;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.RestrictTo;
+
+import java.util.Arrays;
+
+/**
+ * The Power characteristic is used to represent a measure of power in units of watts.
+ */
+public class Power extends DevicePropertyCharacteristic<Float> {
+
+    @RestrictTo(RestrictTo.Scope.LIBRARY)
+    public Power(@NonNull final byte[] data, final int offset, final int length) {
+        super(data, offset, length);
+        int bits = 0;
+        byte[] bytes = Arrays.copyOfRange(data, offset, offset + length);
+        for (byte b : bytes) bits = (bits >> 8) | (((int) b & 0xFF) << 16);
+        if (bits == 0xFFFFFF) this.value = null;
+        else this.value = ((float) bits) / 10.0f;
+    }
+
+    /**
+     * Power characteristic
+     *
+     * @param power Power in units of watts.
+     */
+    public Power(final float power) {
+        this.value = power;
+    }
+
+    @NonNull
+    @Override
+    public String toString() {
+        return this.value + " W";
+    }
+
+    @Override
+    public int getLength() {
+        return 3;
+    }
+
+    @Override
+    public byte[] getBytes() {
+        byte[] bytes = {(byte) 0xFF, (byte) 0xFF, (byte) 0xFF};
+        if (this.value != null) {
+            int value = (int) (this.value * 10.0f + 0.5d);
```

Comment:
Why mixing types 10.0 (float) and 0.5 (double)?

Reply:
Fixed copy&paste error